src/main/java/org/parabot/core/parsers/randoms/PublicRandoms.java

```diff
@@ -46,12 +45,14 @@ public void parse() {
 
     private void download() {
         try {
-            File random = new File(Directories.getCachePath() + "/randoms.jar");
+            File random = new File(Directories.getCachePath() + File.separator + "randoms.jar");
             if (random.exists()) {
                 Core.verbose("Public random dependency already exists, no need to download it...");
                 return;
             }
-            String downloadLink = Configuration.GET_RANDOMS;
+
+            String downloadLink = ((Configuration.BOT_VERSION.isNightly()) ? Configuration.GET_RANDOMS + "?stable=false" : Configuration.GET_RANDOMS);
```

Comment:
We've a Configuration value for `?stable=false`, not sure if it's client only.
If it's client only, add it to the internal API configuration.